controller/controller.js

```diff
@@ -42,6 +42,29 @@ const getTopics = (request, response) => {
     }).catch(next)
   }
 
+  const postComments = (req, res, next) => {
+    const newComment = req.body;
+    console.log(newComment, "<<<< new comment")
+    insertComment(newComment).then((comment) => {
+        console.log(comment, "<<<<<<<<<<")
+        res.status(201).send({comment})
+    })
+  }
+
+  const deleteCommentById = (req, res, next) => {
+    const { comment_id } = req.params;
+    removeCommentById(comment_id).then((result) => {
+            res.status(204).send();      
+    }).catch(next)
+  };
+
+  const getUsers = (req, res, next) => {
+    selectUsers().then((user)=> {
+        res.status(200).send({user})
+    })
+  }
```

Comment:
Nice, but this should still have a .catch(...) incase something unexpected goes wrong